examples/gallery/lines/linefronts.py

```diff
@@ -0,0 +1,84 @@
+"""
+Line fronts
+-----------
+
+Using the :meth:`pygmt.Figure.plot` method you can draw a so-called
+*front* which allows to plot specific symbols distributed along a line
+or curve. Typical use cases are weather fronts, fault lines,
+subduction zones, and more.
+
+A front can be drawn by passing **f**\[±]\ *gap*\[/*size*] to the ``style`` parameter where *gap*
+defines the distance gap between the symbols and *size* the symbol size.
+If *gap* is negative, it is interpreted to mean the number of symbols
```

Comment:
Please wrap this line to similar lengths (ideally less than 80 characters).

Reply:
Done! Thanks 😉